src/views/dialogs/WithdrawalGateDialog.tsx

```diff
@@ -0,0 +1,100 @@
+import styled, { AnyStyledComponent } from 'styled-components';
+
+import { ButtonAction, ButtonType } from '@/constants/buttons';
+import { STRING_KEYS } from '@/constants/localization';
+import { useStringGetter, useURLConfigs } from '@/hooks';
+import { LinkOutIcon } from '@/icons';
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { Button } from '@/components/Button';
+import { Dialog } from '@/components/Dialog';
+import { Icon, IconName } from '@/components/Icon';
+
+type ElementProps = {
+  setIsOpen: (open: boolean) => void;
+  isTransfer?: boolean;
```

Comment:
consider: it sort of makes more sense in my head to just have a `title` prop (since that's all `isTransfer` is currently used for) - or if we envision it getting more complex, I mildly prefer `transferType = transfer | withdrawal` because it more clearly outlines the alternatives

Reply:
I think I will implement the transferType suggestion you had. That way the title strings can be defined in the WithdrawalGateDialog instead of wherever the dialog is triggered.